docs/src/components/CombinationNew.js

```diff
@@ -0,0 +1,66 @@
+// @flow strict
+import { Flex } from 'gestalt';
+import React, { type Node } from 'react';
+import MainSectionCard from './MainSectionCard.js';
+
+type Props = {
+  children: (Object, number) => Node, // flowlint-line unclear-type:off
+  shaded?: boolean,
+  ...
```

Comment:
I have some ideas for how we could close this type and tighten up the passed-through types, but that shouldn't be a blocker here. This is totally fine for now.